tests/e2e/e2e.go

```diff
@@ -19,36 +19,40 @@ func generateTimeBasedName(prefix string) string {
 }
 
 func getActiveElementFromCommandOutput(command string) string {
-	result := runCmd(command + " | sed -n '1!p' | awk 'FNR==2 { print $2 }'")
+	result := runCmdShouldPass(command + " | sed -n '1!p' | awk 'FNR==2 { print $2 }'")
 	return strings.TrimSpace(result)
 }
 
-func runCmd(cmdS string) string {
+// runCmd runs a command
+// and returns the stdout, stderr and exitcode
+func runCmd(cmdS string) (string, string, int) {
```

Comment:
To minimize impact on client code (and potential errors by developers being used to the old `runCmd` function, this new function should be renamed differently (perhaps `internalRunCmd`?) and the name `runCmd` should be kept to cover the original cases. Does that make sense?

Reply:
The real impact on test code will not hurt us a lot IMO unless there is a major functionality change. We can ignore the impact for changing function name and of course handling of ```stderr``` in test suite. The current name change in the long run will bring some good test experience.

Reply:
OK, the important thing was that the old name shouldn't be used for a function that's different since it might cause people to use it improperly thinking it was still as before.